pkg/controller/testing/mock_event_recorder.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2018 The Knative Authors
```

Comment:
@Harwayne should we change Copyright to 2019? And also in the hack boilerplate?

Reply:
Sure.